apps_wishlist.md

```diff
@@ -1,175 +1,309 @@
 # Apps wishlist
 
-<div class="alert alert-info">Before to add an app in wishlist please check your app is not already in apps list: see the <a href="/apps">apps list</a></div>
-
 The following list is a compiled wishlist of applications that would be nice-to-have.
 
-[Edit this list](/write_documentation) to add your own favorite app, or learn to [package apps](/packaging_apps) yourself.
+You can [contribute to this list by adding something you'd like to be packaged](/write_documentation), or learn more on [how to package applications](/packaging_apps) to create a new package or improve a package draft.
+
+<div class="alert alert-info">Before adding an app in this wishlist, please check your app is not <a href="/apps">already available in Yunohost's app catalog</a></div>
+
+| Name   | Description | Upstream | Package draft |
+| ------ | ----------- | -------- | ------------- |
+| 10er10 |  | [Upstream](https://github.com/dready92/10er10) |  |
+| [abantecart](https://abantecart.com) | Create a E-commerce Website |  | [Package Draft](https://github.com/YunoHost-Apps/abantecart_ynh) |
+| adguard |  | [Upstream](https://github.com/AdguardTeam/AdGuardHome) | [Package Draft](https://github.com/YunoHost-Apps/adguard_ynh) |
+| adhocserver | An adhoc Server for the sony PSP |  | [Package Draft](https://github.com/matlink/adhocserver_ynh) |
```

Comment:
```suggestion
| adhocserver | An ad hoc server for the Sony PSP |  | [Package Draft](https://github.com/matlink/adhocserver_ynh) |
```
Not 100% sure this isn't the "adhoc party", but as per https://manuals.playstation.net/document/en/psp/current/settings/adhoc.html